library/Network/IPFS/Client/Error/Types.hs

```diff
@@ -0,0 +1,8 @@
+module Network.IPFS.Client.Error.Types (ErrorBody (..)) where
+import Network.IPFS.Prelude
+
+data ErrorBody = ErrorBody {message :: String}
+instance FromJSON ErrorBody where
+  parseJSON = withObject "ErrorBody" \obj -> do
+    message    <- obj .: "Message"
+    return <| ErrorBody {..}
```

Comment:
No need to change, but as a general readability practice, a bit more space is both helpful and more consistent as you get more imports and instances in a module

```haskell
module Network.IPFS.Client.Error.Types (ErrorBody (..)) where

import Network.IPFS.Prelude

data ErrorBody = ErrorBody { message :: String }

instance FromJSON ErrorBody where
  parseJSON = withObject "ErrorBody" \obj -> do
    message    <- obj .: "Message"
    return <| ErrorBody {..}
```

Reply:
Also, you may be able to lose that `<|` after the `return`

```haskell
return ErrorBody {..}
```